docs/resources/notification_policy.md

```diff
@@ -130,6 +130,7 @@ Optional:
 - `status` (Set of String) Status to alert on.
 - `target_hostname` (Set of String) Target host to alert on for dos.
 - `target_zone_name` (Set of String) Target domain to alert on.
+- `tunnel_id` (Set of String) Tunnel IDs to alert on.
```

Comment:
thanks for the PR! unfortunately, the file you've edited is a generated artifact of the documentation process. to see how to make changes to the documentation, you're best to check out https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/registry-documentation.md

Reply:
That explains why there were a couple more changes when I ran `make docs`. Thanks for fixing it, and I'll know better for next time!